conf/messages

```diff
@@ -39,6 +39,10 @@ global.banner.text = This is a new service – your
 global.banner.feedback = feedback
 global.banner.helpimprove = will help us to improve it.
 
+phase.banner.before=This is a new service – your
```

Comment:
Why are these messages being added to your own messages file? They should be imported via the messages file in play-frontend-hmrc.

Reply:
https://github.com/hmrc/play-frontend-hmrc/blob/389a9a424ae64e9f0695401ce97be91e13a5fb19/src/main/resources/messages#L2